vendor_server.cc

```diff
@@ -39,49 +25,48 @@ using grpc::ServerBuilder;
 using grpc::ServerContext;
 using grpc::Status;
 using grpc::StatusCode;
-using supplyfinder::HelloRequest;
-using supplyfinder::HelloReply;
-using supplyfinder::Vendor;
 using supplyfinder::FoodID;
 using supplyfinder::InventoryInfo;
+using supplyfinder::Vendor;
 
 // Logic and data behind the server's behavior.
 class VendorServiceImpl final : public Vendor::Service {
-  public:
-  VendorServiceImpl () {
+ public:
+  VendorServiceImpl() {
     /*
-     * Dummy database which contains :
-     * id 1: price 19.5, quantity 10
-     * id 3: price 2.99, quantity 2
+     * Randomly generate inventory information
+     * Each vendor has five items, with price [0, 20.0] and quantity [0, 99]
      */
+    std::vector<int> indices = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9};
+    std::random_device rd;
+    std::mt19937 g(rd());
+    std::shuffle(indices.begin(), indices.end(), g);
```

Comment:
nit: it might be cool to shuffle the indices for each vendor, so they all have different foods :)

Reply:
Yeah the indices are shuffled separately when constructing each vendor :)